packages/grafana-ui/src/components/BigValue/BigValueLayout.tsx

```diff
@@ -56,12 +63,16 @@ export abstract class BigValueLayout {
     }
   }
 
-  getTitleStyles(): CSSProperties {
+  getTitleStyles(theme: GrafanaTheme2): CSSProperties {
     const styles: CSSProperties = {
       fontSize: `${this.titleFontSize}px`,
       lineHeight: LINE_HEIGHT,
     };
 
+    if (this.props.parentOrientation === VizOrientation.Horizontal && this.justifyCenter) {
+      styles.paddingRight = this.props.theme.spacing(1);
```

Comment:
what would something font-dependant look like, like `1ch` (or maybe like `0.75ch`) instead?

Reply:
Put differently, is using fixed pixels, where the gap will remain constant size regardless of the surrounding font size, the right approach?

Reply:
Good point! Here is what a very small and very large one look like, the top line is with 8px padding the bottom one is with 0.75ch. You barely notice the difference on the big one but on the small one the `ch` definitely looks better:
<img width="718" alt="Screenshot 2022-09-16 at 14 42 15" src="https://user-images.githubusercontent.com/100691367/190652938-3f415399-d38c-4b01-a92b-24be310ff8c5.png">
<img width="120" alt="Screenshot 2022-09-16 at 14 42 22" src="https://user-images.githubusercontent.com/100691367/190652945-ce6fde72-d97f-43eb-98ba-9ba4f9ab0f65.png">